accounts/store_test.go

```diff
@@ -105,6 +105,188 @@ func assertEqualAccounts(t *testing.T, expected,
 	actual.LastUpdate = actualUpdate
 }
 
+// TestAccountUpdateMethods tests that all the Store methods that update an
+// account work correctly.
+func TestAccountUpdateMethods(t *testing.T) {
```

Comment:
Similar to the other PR, we should test the behaviour for an unknown account id :).